apps/website-25/src/components/homepage/HomeHeroContent.tsx

```diff
@@ -0,0 +1,25 @@
+import {
+  HeroH1,
+  HeroH2,
+} from '@bluedot/ui';
+import Rive, { Fit, Layout } from '@rive-app/react-canvas';
+import clsx from 'clsx';
+
+const HomeHeroContent: React.FC<{ className?: string }> = ({ className }) => (
+  <div className={clsx('pb-5', className)}>
+    <div className="hero-section__animation-container w-[350px] max-w-[80vw] aspect-4/3 mx-auto">
+      <Rive
+        src="/animations/herobluedot_hero.riv"
+        layout={new Layout({ fit: Fit.Fill })}
+      />
+    </div>
```

Comment:
Unfortunately there is no way to:
1. [Change the speed of the animation](https://rive.app/community/forums/support/fsnLLkXUaA62/how-do-i-control-the-animation-speed-at-runtime/ftFXEAm7CVmE)
2. [Use the natural aspect ratio of the animation](https://github.com/rive-app/rive-ios/issues/323)